src/Console/Command/DocumentationCommand.php

```diff
@@ -100,7 +100,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         foreach ($setDefinitions as $name => $definition) {
             $path = $locator->getRuleSetsDocumentationFilePath($name);
-            $paths[$name] = $path;
+            $paths[$path] = $definition;
```

Comment:
Can you describe this change ?

Reply:
I needed to pass rulesets' full definition to `RuleSetDocumentationGenerator::generateRuleSetsDocumentationIndex()` in order to determine if it's deprecated. Previously only name+path was passed to the doc generator so I would need to re-instantiate the rule set with `RuleSets::getSetDefinitions()` or something like that. I thought it's better to pass it from top to bottom since we already had that information.